pymatgen/analysis/chemenv/coordination_environments/coordination_geometries.py

```diff
@@ -73,8 +73,7 @@ def __str__(self):
 
 
 class ExplicitPermutationsAlgorithm(AbstractChemenvAlgorithm):
-    """
-    Class representing the algorithm doing the explicit permutations for the calculation of
+    """Algorithm doing the explicit permutations for the calculation of
```

Comment:
Thanks for these docstring improves though. I was think the same. 

The keyword `class` should already be self-documenting and no need to repeat it's "Class for some purpose."

Also as per [PEP-8](https://peps.python.org/pep-0008/) the docstring should usually start with a brief high-level summary on the first line (the line with ```), so I totally support your changes here!